doc/2/guides/elasticsearch/v8/index.md

```diff
@@ -0,0 +1,156 @@
+---
+code: false
+type: page
+order: 100
+title: Elasticsearch 8 | Develop on Kuzzle | Guide | Core
+meta:
+  - name: description
+    content: Extend Kuzzle API with controllers and actions
+  - name: keywords
+    content: Kuzzle, Documentation, kuzzle write pluggins, General purpose backend, iot, backend, opensource,  API Controllers
+---
+
+# Elasticsearch 8
+
+<SinceBadge version="2.30.0"/>
+
+Kuzzle uses Elasticsearch as a [NoSQL document store](/core/2/guides/main-concepts/data-storage).
+
+With Kuzzle, customers **can directly access data stored in the database** as long as they have the rights to do so.
+
+Kuzzle exposes the [Elasticsearch Query Language](/core/2/guides/main-concepts/querying) in a secure way. It is therefore possible to **take full advantage of the possibilities of Elasticsearch** with boolean queries, aggregations, special fields, etc.
+
+The support of Elasticsearch 8 has been introduced in Kuzzle 2.30.0.
+
+The choice has been made to keep Kuzzle compatible to avoid breaking changes around the support of ES8. 
+
+We wanted to allow the user to **opt-in** for this feature, so no modification is needed on your behalf if you want to keep using Elasticsearch 7.
+
+The default major version of Elasticsearch will be 7 until Kuzzle v3.
+
+The new configuration key to change the version supported is available under:  
+
+```json
+{
+  "services": {
+    "storageEngine": {
+      "majorVersion": 8
+    }
+  }
+}
+```
+
+:::warning
+You can not set the `majorVersion` key to 8 if you are using a version of Kuzzle that does not support it. 
+:::
+
+:::info
+Kuzzle cannot connect to both Elasticsearch 7 and Elasticsearch 8 at the same time.  
+:::
```

Comment:
I guess it's not ::: but ``` here on these 7 lines

Reply:
This is part of the kuzzle documentation paradigm :)